app/Helper.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+class Helper
+{
+
+    public static function transformDate($date, $format = 'd-m-Y')
```

Comment:
Add php doc.
Rename to `formatToDateTime`
I think we can omit this parameter in favour of using config param as a single place of configuring date-time format.

Reply:
Done